app/views/pages/netlify_config.yaml.erb

```diff
@@ -97,6 +97,42 @@ collections:
     fields:
       - label: "Title"
         name: "title"
+      - label: "Hero"
+        name: "hero"
+        widget: object
+        collapsed: true
+        fields:
+          - label: "Show hero"
+            name: enabled
+            widget: boolean
+            default: false
+          - label: "Title"
+            name: title
+            widget: string
```

Comment:
[suggestion] I think this needs to be a `textarea` widget. It looks like it isn't possible to use the editor to put a newline into the title.